### PR TITLE
New topnav and footer

### PR DESCRIPTION
**Pull request by cjyabraham**

### CODE REVIEW

File: pytorch-domains.html [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+---
+
+<div class="jumbotron jumbotron-fluid contributor-jumbotron">
+  <div class="contributor-jumbo-text container">
+    <h1>PyTorch Domains</h1>
+    <h2 class="lead">Learn about the various parts of the PyTorch ecosystem</h2>
+
+  </div>
+</div>
+
```
Comment (svekars) [inline pytorch-domains.html:16]: Can we add another card here? This library was just released today:
```
<h4>torchtune</h4>
<p>A PyTorch-native library designed for fine-tuning large language models (LLMs). torchtune provides supports the full fine-tuning workflow and offers compatibility with popular production inference systems. <p>
```

File: _includes/main_menu.html [hunk 2]
```diff
@@ -1,144 +1,165 @@
...
-          <a class="nav-dropdown-item" href="{{ site.baseurl }}/hub">
-            <span class=dropdown-title>Models (Beta)</span>
-            <p>Discover, publish, and reuse pre-trained models</p>
-          </a>
         </div>
       </div>
     </li>
 
+    <li class="main-menu-item">
+      <a href="{{ site.baseurl }}/join" data-cta="join">
```
Comment (svekars) [inline _includes/main_menu.html:154]: If you hover over Join and GitHub it uses a cursor while other items use arrow - we should have it consistently.

File: _includes/main_menu.html [hunk 3]
```diff
@@ -1,144 +1,165 @@
...
       <div id="docsDropdownButton" data-toggle="resources-dropdown" class="resources-dropdown">
-        <a class="resource-option {% if (current[1] == {{ site.baseurl }} or current[1] == 'foundation' or current[1] == 'join' or current[1] == 'edge' or current[1] == 'executorch-overview') %}with-down-white-arrow{% else %}with-down-arrow{% endif %}">
+        <a class="with-down-arrow">
           Docs
         </a>
         <div class="resources-dropdown-menu">
-          <a class="nav-dropdown-item" href="{{ site.baseurl }}/docs">
+          <a class="nav-dropdown-item" href="https://pytorch.org/docs">
             <span class="dropdown-title docs-title">PyTorch</span>
             <p></p>
```
Comment (svekars) [inline _includes/main_menu.html:97]: Should we add text here for consistency with other dropdowns?
```suggestion
            <p>Explore the documentation for comprehensive guidance on how to use PyTorch.</p>
```

File: _includes/main_menu.html [hunk 4]
```diff
@@ -1,144 +1,165 @@
...
-          </a>
-          <a class="nav-dropdown-item" href="{{ site.baseurl }}/text">
-            <span class="dropdown-title docs-title">torchtext</span>
-            <p></p>
-          </a>
-          <a class="nav-dropdown-item" href="{{ site.baseurl }}/vision">
-            <span class="dropdown-title docs-title">torchvision</span>
+          <a class="nav-dropdown-item" href="{{ site.baseurl}}/pytorch-domains">
+            <span class="dropdown-title docs-title">PyTorch Domains</span>
             <p></p>
```
Comment (svekars) [inline _includes/main_menu.html:101]: Same here
```suggestion
            <p> Read the PyTorch Domains documentation to learn more about domain-specific libraries.</p>
```

File: _includes/main_menu.html (hunk 2 above)
Comment (cjyabraham) [reply _includes/main_menu.html:154]: "Join" and the GitHub icon are clickable while the other top menu items are not that's why the mouse pointer behaves this way.

File: pytorch-domains.html [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
```
Comment (svekars) [inline pytorch-domains.html:16]: we should add a couple more here:
**TorchRL**
TorchRL is an open-source, Python-first Reinforcement Learning library for PyTorch with a focus on high modularity and good runtime performance, providing low and high-level RL abstractions and reusable functionals for cost functions, returns, and data processing.

**TensorDict**
TensorDict is a dictionary-like class that simplifies operations on batches of tensors, enhancing code readability, compactness, and modularity by abstracting tailored operations and reducing errors through automatic operation dispatching on the leaves.

Comment (cjyabraham) [reply pytorch-domains.html:16]: That's done.

**Pull request merged**